public/locales/en/panel-nightsky.json

```
@@ -0,0 +1,249 @@
+      "look-north": "Look North",
+      "look-east": "Look East",
+      "look-south": "Look South",
+      "look-west": "Look West"
+    },
+    "horizon": {
+      "title": "Horizon",
+      "look-horizion": "Look at Horizon",
+      "level-horizon": "Level Horizon",
+      "look-up": "Look Up"
+    }
+  },
+  "solar-system": {
+    "trails": {
+      "title": "Trails",
+      "buttons": {
+        "hide-trails": "Hide ALL Trails",
+        "show-planet-trails": "Show Planet Trails"
+      }
+    },
+    "labels": {
+      "title": "Labels",
+      "buttons": {
+        "show-labels": "Show Labels",
+        "hide-labels": "Hide Labels"
+      }
+    },
+    "plaents": {
+      "title": "Planets",
+      "buttons": {
+        "show-planets": "Show Night Sky Planets",
+        "hide-planets": "Hide Night Sky Planets"
+      }
+    },
+    "moon": {
+      "title": "Moon",
+      "buttons": {
+        "default-scale": "Default Moon Scale (1x)",
+        "2x-scale": "Enlarge Moon (2x)",
+        "4x-scale": "Enlarge Moon (4x)"
+      },
+      "aria-labels": {
+        "increase-scale": "Increase moon scale",
+        "decrease-scale": "Decrease moon scale"
+      },
+      "error": {
+        "moon-scale-settings": "Could not find Moon scale settings",
+        "moon-shading-settings": "Could not find Moon Perform Shading settings"
+      },
+      "show-phase-input": {
+        "title": "Show Phase",
+        "tooltip": "Uncheck this to show the full moon always. This is equivalent to 'Perform Shading' property on the Moon."
+      }
+    }
+  },
+  "stars": {
+    "visibility": {
+      "title": "Visibility",
+      "input": {
+        "show-stars": "Show Stars",
+        "show-during-day": {
+          "label": "Show During Day",
+          "tooltip": "Check this box to show the stars during daytime"
+        }
+      }
+    },
+    "labels": {
+      "title": "Labels",
+      "input": {
+        "show-labels": "Show Labels",
+        "show-alternate-labels": "Show Alternate Labels"
+      }
+    },
+    "appearance": {
+      "title": "Apperance",
+      "warning": {
+        "title": "Only Night Sky Profile",
+        "description": "The buttons below only work in the 'nightsky' profile. Use the profile editor to edit the actions and adjust the values for your dome."
+      },
+      "buttons": {
+        "default-settings": "Default Settings",
+        "point-like-stars": "More Point Like Stars"
+      }
+    }
+  },
+  "sun": {
+    "glare": {
+      "title": "Glare",
+      "buttons": {
+        "show-glare": "Sow Glare",
+        "hide-glare": "Hide Glare"
+      }
+    },
+    "size": {
+      "title": "Size",
+      "buttons": {
+        "default-size": "Default Angular Size",
+        "large-size": "Large Angular Size",
+        "extra-large-size": "Extra Large Angular Size"
+      },
+      "aria-labels": {
+        "increase-size": "Increase angular size",
+        "decrease-size": "Decrease angular size"
```

```suggestion
        "increase-size": "Increase Angular Size",
        "decrease-size": "Decrease Angular Size"
```

?  For consistency with the other labels

---

Alright!